coding-guides/tracking-used-invites.md

```diff
@@ -20,11 +20,11 @@ While the below is a fair approximation of invite tracking, it's still not perfe
 
 ## Caching Invites
 
-The first part is to fetch all the invites and keep them cached in an Map. This is done in the `ready` event. First, however, we must ensure that the cache is initialized _outside_ of the ready event.
+The first part is to fetch all the invites and keep them cached in a [Collection](https://anidiots.guide/understanding/collections/). This is done in the `ready` event. First, however, we must ensure that the cache is initialized _outside_ of the ready event.
 
 ```javascript
 // Initialize the invite cache
-const invites = new Map();
+const invites = new Collection();
```

Comment:
Using Maps is perfectly fine but Collections are pretty much the same but with extra methods added
like find and filter, which can be used when expanding the command and we already explain collections so why not